test/e2e/common/configmap_volume.go

```diff
@@ -637,8 +634,14 @@ func doConfigMapE2EWithoutMappings(f *framework.Framework, uid, fsGroup int64, d
 		},
 	}
 
-	if userID != 0 {
-		pod.Spec.SecurityContext.RunAsUser = &userID
+	if asUser {
+		if framework.NodeOSDistroIs("windows") {
+			userName := "ContainerUser"
+			pod.Spec.SecurityContext.WindowsOptions = &v1.WindowsSecurityContextOptions{RunAsUserName: &userName}
+		} else {
+			userID := int64(1000)
```

Comment:
Hard-coding the user ID to 100 is a little bit weird. Making it a constant (e.g., `nonRootTestUserID`) may make it more readable.

Reply:
Done